Tests/aweXpect.Reflection.Tests/Filters/ThatConstructor.HasParameter.Tests.cs

```diff
@@ -0,0 +1,104 @@
+using System.Reflection;
+
+namespace aweXpect.Reflection.Tests.Filters;
+
+public sealed partial class ThatConstructor
+{
+	public sealed class HasParameter
+	{
+		public sealed class Tests
+		{
+			[Fact]
+			public async Task HasParameterByType_WhenParameterExists_ShouldSucceed()
+			{
+				ConstructorInfo constructorInfo = typeof(TestClass).GetConstructor([typeof(int), typeof(string)])!;
+
+				async Task Act()
+					=> await That(constructorInfo).HasParameter<int>();
+
+				await That(Act).DoesNotThrow();
+			}
+
+			[Fact]
+			public async Task HasParameterByType_WhenParameterDoesNotExist_ShouldFail()
+			{
+				ConstructorInfo constructorInfo = typeof(TestClass).GetConstructor([typeof(string)])!;
+
+				async Task Act()
+					=> await That(constructorInfo).HasParameter<int>();
+
+				await That(Act).ThrowsException()
```

Comment:
Always verify the exception type (`XunitException`) and the exception message in tests when it throws an exception.